### PR TITLE
Update to Lisk Service v0.5 docs - Closes #1203

### LINKED ISSUE #1203: Update Service docs to v0.5.0
### Description
The Lisk Service documentation needs to be updated to reflect the latest release of 0.5.0:
https://github.com/LiskHQ/lisk-service/releases/tag/v0.5.0-beta.0

### Additional information
Docker & README updates: https://github.com/LiskHQ/lisk-service/pull/792/files
See also: https://github.com/LiskHQ/lisk-docs/pull/1167

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1203

### CODE REVIEW

File: modules/ROOT/pages/setup/source.adoc [hunk 1]
```diff
@@ -177,15 +180,16 @@ sudo apt update
...
 macOS::
 +
 --
 [source,bash]
 ----
 brew tap homebrew/services
-brew install mysql@5.7
-brew services start mysql@5.7
+brew install mysql@8.0
+brew services start mysql@8.0
```
Comment (MichalTuleja) [inline modules/ROOT/pages/setup/source.adoc:192]: I think we need to mention the password authentication as it is described in the article:


>Authentication
>There’s an issue to authenticate, where I did received the following error:
caching_sha2_password' cannot be loaded: dlopen(/usr/local/lib/plugin/caching_sha2_password.so, 2): image not found
>So what I did, I change the default_authentication_plugin using mysql_native_password.
>Open up my.cnf — mine located at /usr/local/etc/my.cnf
>Add the following at the end of the file:
>
>```
>default_authentication_plugin=mysql_native_password
>```

Source: https://medium.com/@nasrulhazim/mysql-8-2ff3ab396f20

Comment (Tschakki) [reply modules/ROOT/pages/setup/source.adoc:192]: added a note 👍

**Review by MichalTuleja — DISMISSED** [review]
LGTM 👍

File: modules/ROOT/pages/configuration/docker.adoc [hunk 2]
```diff
@@ -12,20 +12,128 @@ Mona Bärenfänger <mona@lightcurve.io>
...
+Copy the example configuration options to `lisk-service/.env`:
+
+.lisk-service/
+[source,bash]
+----
+cp docker/example.env .env
+vim .env
+----
+
+Open the `.env` file to change thee default configuration options of Lisk Service:
```
Comment (chris529) [inline modules/ROOT/pages/configuration/docker.adoc]:
```suggestion
Open the `.env` file to change the default configuration options of Lisk Service:
```

File: modules/ROOT/pages/configuration/source.adoc [hunk 3]
```diff
@@ -16,7 +16,7 @@ Mona Bärenfänger <mona@lightcurve.io>
 
 == The ecosystem config files
 
-The {url_pm2_ecosystem_file}[ecosystem file^] provides the possibility to configure behavior, options, environment variables and the logs files of each application with {url_pm2}[PM2^].
+The {url_pm2_ecosystem_file}[ecosystem file^] provides the possibility to configure behavior, options, environment variables and the logs files of Lisk Service with PM2.
```
Comment (chris529) [inline modules/ROOT/pages/configuration/source.adoc]:
```suggestion
The {url_pm2_ecosystem_file}[ecosystem file^] provides the possibility to configure behavior, options, environment variables, and the logs files of Lisk Service with PM2.
```

File: modules/ROOT/pages/configuration/source.adoc [hunk 4]
```diff
@@ -26,12 +26,17 @@ pm2 start ecosystem.mainnet.config.js
 
 === Default configurations
 
-There are 2 default ecosystem files in the root folder of `lisk-service`:
+The default ecosystem file `ecosystem.core3.config.js` is in the root folder of `lisk-service`:
```
Comment (chris529) [inline modules/ROOT/pages/configuration/source.adoc]:
```suggestion
The default ecosystem file `ecosystem.core3.config.js` is located in the root folder of `lisk-service`:
```

File: modules/ROOT/pages/configuration/source.adoc [hunk 5]
```diff
@@ -26,12 +26,17 @@ pm2 start ecosystem.mainnet.config.js
...
+It contains a configuration to connect to a local Lisk Core node.
 
-* `ecosystem.mainnet.config.js` contains a configuration to connect to the Lisk Mainnet (default).
-* `ecosystem.testnet.config.js` contains a configuration to connect to the Lisk Testnet.
+This config will be used by default when starting Lisk Seervice with:
```
Comment (chris529) [inline modules/ROOT/pages/configuration/source.adoc]:
```suggestion
This config will be used by default when starting Lisk Service with the following:
```

File: modules/ROOT/pages/index.adoc [hunk 6]
```diff
@@ -64,6 +64,13 @@ This provides users with a **central point of data access** that never breaks ex
 Its main purpose is to provide enriched data from the Lisk SDK API.
 This component is designed to provide high availability, efficient and reliable access to the Lisk SDK API.
```
Comment (chris529) [inline modules/ROOT/pages/index.adoc]:
```suggestion
This component is designed to provide high availability, together with efficient and reliable access to the Lisk SDK API.
```

File: modules/ROOT/pages/index.adoc [hunk 7]
```diff
@@ -64,6 +64,13 @@ This provides users with a **central point of data access** that never breaks ex
...
 
+|*Market*
+|The Market service allows price data retrieval.
+It supports multiple sources to keep the current Lisk token price up-to-date and available to the clients in real time.
```
Comment (chris529) [inline modules/ROOT/pages/index.adoc]:
```suggestion
It supports multiple sources to maintain the current up-to-date Lisk token price, and ensures this is available to the clients in real time.
```

File: modules/ROOT/pages/references/rpc-api.adoc [hunk 8]
```diff
@@ -1146,3 +1146,144 @@ No parameters.
...
+[cols="1,1,2", options="header"]
+.The Lisk Service microservices
+|===
+|Parameter |Type |Validation
+
+|source
+|String
+|`/[A-z]+/`   \| `*`
+
+Retrieves all sources by default
```
Comment (chris529) [inline modules/ROOT/pages/references/rpc-api.adoc]:
```suggestion
Retrieves all sources by default.
```

File: modules/ROOT/pages/setup/source.adoc [hunk 9]
```diff
@@ -177,16 +180,49 @@ sudo apt update
...
+****
+If you encounter issues to authenticate, and received the following error:
+
+ caching_sha2_password' cannot be loaded: dlopen(/usr/local/lib/plugin/caching_sha2_password.so, 2): image not found
+
+Try the following: Change the `default_authentication_plugin` using `mysql_native_password`.
+
+Open up `my.cnf` .
+
+If you not sure where your `my.cnf`, use the following command:
```
Comment (chris529) [inline modules/ROOT/pages/setup/source.adoc]:
```suggestion
If you are unsure where your `my.cnf`, use the following command:
```

File: modules/ROOT/pages/setup/source.adoc [hunk 10]
```diff
@@ -177,16 +180,49 @@ sudo apt update
...
+
+ $ mysql --verbose --help | grep my.cnf
+
+Add the following at the end of the file:
+
+ default_authentication_plugin=mysql_native_password
+
+Save and exit.
+
+Next, you need to login via terminal:
```
Comment (chris529) [inline modules/ROOT/pages/setup/source.adoc]:
```suggestion
Next, you need to login via the terminal:
```

**Review by chris529 — COMMENTED** [review]
Looks good, just a few minor points 👍

File: modules/ROOT/pages/setup/source.adoc [hunk 11]
```diff
@@ -177,16 +180,49 @@ sudo apt update
...
+****
+If you encounter issues to authenticate, and received the following error:
+
+ caching_sha2_password' cannot be loaded: dlopen(/usr/local/lib/plugin/caching_sha2_password.so, 2): image not found
+
+Try the following: Change the `default_authentication_plugin` using `mysql_native_password`.
+
+Open up `my.cnf` .
+
+If you are unsure where your `my.cnf`, use the following command:
```
Comment (chris529) [inline modules/ROOT/pages/setup/source.adoc]:
```suggestion
If you are unsure where to find your `my.cnf`, use the following command:
```

**Review by chris529 — DISMISSED** [review]
LGTM! Only 1 minor point 👍

**Review by chris529 — APPROVED** [review]
LGTM!

**Pull request merged**